starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,72 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <title>Apple Pie Recipe</title>
-</head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <head>
+    <title>Apple Pie Recipe</title>
+    <link
+      rel="stylesheet"
+      href="/starter-code/apple-pie-starter-code/css/main.css"
+    />
+  </head>
+  <body class="body-align">
+    <div class="image-manipulation-upper">
+      <h1 id="title-color" class="centered">Apple Pie</h1>
+    </div>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+    <div class="page-margin">
+      <div>
+        <p>
+          This was my grandmother's apple pie recipe. I have never seen another
+          one quite like it. It will always be my favorite and has won me
+          several first place prizes in local competitions. I hope it becomes
+          one of your favorites as well!
+        </p>
+        <div class="image-manipulation-lower">
+          <img
+            class="image-resize"
+            src="/starter-code/apple-pie-starter-code/images/recipe-info.png"
+            alt="About recipe"
+          />
+        </div>
+        <hr />
+      </div>
+      <div>
+        <h2>Ingredients</h2>
+        <p class="indent">
+          1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter
+          3 tablespoons all-purpose flour 1/4 cup water 3 tablespoons
+          all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
+          brown sugar 8 Granny Smith apples - peeled, cored and sliced
+        </p>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+        <div class="image-manipulation-lower">
+          <img
+            class="image-resize"
+            src="/starter-code/apple-pie-starter-code/images/cooking-info.png"
+            alt="Cooking time"
+          />
+        </div>
+      </div>
+      <hr />
 
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
+      <h2>Directions</h2>
+      <div class="indent">
+        <p>
```

Comment:
same here. you want to create a list (this time it should have a specific order) so use `ol` and `li` tags.